src/domain/feedbackForm/FeedbackForm.tsx

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+import { TextInput, TextArea, Checkbox, Button, IconUpload } from "hds-react";
+
+import A from "../../common/components/a/A";
+import PlainList from "../../common/components/plainList/PlainList";
+import Text from "../../common/components/text/Text";
+import styles from "./feedbackForm.module.scss";
+
+interface Props {
+  fluid?: boolean;
+}
+
+function FeedbackForm({ fluid = false }: Props) {
+  const [showContactDetailFields, setShowContactDetailFields] = React.useState(
+    false
+  );
+
+  const handleWantReplyToggle = () => {
+    setShowContactDetailFields((value) => !value);
+  };
+
+  const wantReplyValue = showContactDetailFields;
+
```

Comment:
Actually, isn't this a little redundant?

Reply:
It may be, but I am not sure yet. I left this in as an "interface" so that I can more easily understand what separation of concerns I planned for this component. I think this line as a selector of sorts. The component using this value can be agnostic of how it's sourced.

I expect this to be resolved in KUVA-415.